
bool   call
bool   go
int32  station_go
int32  APP_NODE
int32  time
int32  APP_ID
time   call_time

